lib/sorbet-rails/active_record_rbi_formatter.rb

```diff
@@ -135,6 +135,29 @@ def generate_active_record_relation_rbi
           return_type: "Elem",
         )
       end
+
+      class_rbi.create_method(
+        "find",
+        parameters: [Parameter.new("*args", type: "T.untyped")],
+        return_type: "Elem",
+      )
+
+      if Rails.version =~ /^5\.0/
+        item_methods = %w(first second third third_to_last second_to_last last)
+        item_methods.each do |item_method|
+          class_rbi.create_method(
+            item_method,
+            parameters: Rails.version =~ /^5\.0/ ? [Parameter.new("*args", type: "T.untyped")] : [Parameter.new("limit", type: "T.untyped", default: "nil")],
```

Comment:
This seems to be the same check as the outer "if"?

Reply:
You're right, nice catch 👍